boards/toradex/verdin_imx8mm/doc/index.rst

```diff
@@ -0,0 +1,268 @@
+embedded System on Chips (SoCs). The i.MX 8M Mini family consists of the i.MX 8M Mini Quad,
+i.MX 8M Mini QuadLite, i.MX 8M Mini Dual, i.MX 8M Mini DualLite, i.MX 8M Mini Solo, and i.MX
+8M Mini SoloLite. The top-tier i.MX 8M Mini Quad features four Cortex-A53 cores as the main
+processor cluster. The cores provide complete 64-bit Armv8-A support while maintaining seamless
+backwards compatibility with 32-bit Armv7-A software. The main cores run at up to 1.8 GHz for
+commercial graded products and 1.6 GHz for industrial temperature range products.
+
+In addition to the main CPU complex, the i.MX 8M Mini features a Cortex-M4F processor which
+peaks up to 400 MHz. This processor is independent of the main complex and features its own
+dedicated interfaces while still being able to access the regular interfaces. This heterogeneous
+multicore system allows for the running of additional real-time operating systems on the M4 cores
+for time- and security-critical tasks.
+
+- Board features:
+
+  - RAM: 1GB - 2GB (LPDDR4)
+  - Storage:
+
+    - 4GB - 17GB eMMC
+    - 2KB I²C EEPROM
+  - Wireless:
+
+    - WiFi Dual-band 802.11 ac/a/b/g/n 2.4/5 GHz
+    - Bluetooth 5/BLE
+  - USB:
+
+    - 1x USB2.0 Host
+    - 1x USB2.0 OTG
+  - Ethernet Gigabit
+  - Interfaces:
+
+    - MIPI DSI 1x4 Data Lanes
+    - PCIe Gen 2
+    - MIPI CSI-2
+    - SPI
+    - QSPI
+    - UART
+    - I²C
+    - I²S
+    - SD/SDIO/MMC
+    - GPIOs
+    - CAN
+    - ADC
+    - S/PDIF
+  - Debug
+
+    - JTAG 10-pin connector
+
+More information about the board can be found at the
+`Toradex website`_.
+
+Supported Features
+==================
+
+The Zephyr verdin_imx8mm_m4 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | clock_control                       |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | GPIO output                         |
+|           |            | GPIO input                          |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+:zephyr_file:`boards/arm/verdin_imx8mm_m4/verdin_imx8mm_m4_defconfig`.
```

Comment:
File path here needs to be updated

Reply:
This has been addressed in the latest push.